geowebcache/core/src/main/java/org/geowebcache/config/BlobStoreInfo.java

```diff
@@ -70,6 +70,15 @@ public String getId() {
         return id;
     }
 
+    /**
+     * Returns this {@link Info Info}s name. For now, this just returns this BlobStoreInfo's ID.
+     * @return A String representing the name of this Info implementation.
+     */
+    @Override
+    public String getName() {
+        return getId();
+    }
```

Comment:
Given we are switching to getName as the unique identifier, should also mark get/setId as `@Deprecated` (And probably also add a setName to replace setId)

Additionally, changing argument and variable names (mostly in the constructor) from id to name would probably be a good idea as well.